versions/1.1.0-dev.md

```diff
@@ -252,7 +280,7 @@ info:
   title: Apply Traits
   version: 1.0.0
 actions:
-  - target: $.paths[?@.get['x-oai-traits'][?@ == 'paged']].get
+  - target: $.paths.*.get[?(@['x-oai-traits'][?(@ == 'paged')])]
```

Comment:
This change isn't necessary for RFC-9535, but it would increase the compatibility with legacy libraries.